backend/api/elite2Api.js

```diff
@@ -9,7 +9,7 @@ const elite2ApiFactory = client => {
 
   const get = (context, url, resultsLimit) => client.get(context, url, resultsLimit).then(processResponse(context))
 
-  const getStream = (context, url) => client.getStream(context, url).then(processResponse(context))
+  const getStream = (context, url) => client.getStream(context, url)
```

Comment:
copied from notm - had exactly the same problem first time around then :woman_facepalming: